### PR TITLE
100% code coverage

### PR DESCRIPTION
**Pull request by SamVerschueren**
I'm picking this up after AVA had fixed some issues regarding processes not being killed correctly. Made this PR to discuss the code coverage stuff a little bit easier.

It's very hard to read the coverage reports as they are being applied against the transpiled source code. Any idea how we can map that? Or should we just try to map it by hand :)?

// @sindresorhus @kevva @arthurvr

### CODE REVIEW

**Comment by sindresorhus** [comment]
@SamVerschueren Maybe related: https://github.com/bcoe/nyc/issues/53

File: test/test.js [hunk 1]
```diff
@@ -50,6 +51,20 @@ test('generate screenshots', async t => {
...
+	const pageres = new Pageres().src('yeoman.io', ['480x320', '1024x768', 'iphone 5s']);
+
+	await pageres.run();
+
+	pageres.successMessage();
+
+	t.is(stub.firstCall.args[0], '\n✔ Generated 3 screenshots from 1 url and 1 size');
+
+	stub.restore();
+});
```
Comment (sindresorhus) [inline test/test.js]: Too many empty lines. Can you condense this a bit?

Comment (SamVerschueren) [inline test/test.js]: Yeah sure :). What's the preferred style here? Remove all the empty lines? Group const/let declarations and method calls?

```js
test('success message', async t => {
	const stub = sinon.stub(console, 'log');
	const pageres = new Pageres().src('yeoman.io', ['480x320', '1024x768', 'iphone 5s']);
	await pageres.run();
	pageres.successMessage();
	t.is(stub.firstCall.args[0], '\n✔ Generated 3 screenshots from 1 url and 1 size');
	stub.restore();
});
```

**Comment by SamVerschueren** [comment]
Should we wait untill babel is supported by `nyc` or should we try to get 100% coverage with what we have?

File: test/test.js (hunk 1 above)
Comment (sindresorhus) [inline test/test.js]: It's hard to have a strict standard about this stuff because it's usually so ambiguous. I think the above looks good though.

Comment (SamVerschueren) [inline test/test.js]: :+1:

**Comment by sindresorhus** [comment]
@SamVerschueren That's really up to you. 100% coverage isn't very urgent, but if you want to work on it now rather than waiting, feel free.

**Comment by sindresorhus** [comment]
nyc@4, which I just upgraded us to, has Babel support.

**Comment by SamVerschueren** [comment]
Thanks for letting me know!

**Comment by sindresorhus** [comment]
@SamVerschueren Can you fix the merge conflict? I guess we can just merge this one for now and continue on 100% coverage at a later point.

**Comment by SamVerschueren** [comment]
Sorry, didn't find the time to look into babel support for `nyc`.

**Comment by kevva** [comment]
Sweet yo, onto that 100% :).

**Pull request merged**